Chat_App/Chat_App/Assets.xcassets/SendButton.imageset/Contents.json

```diff
@@ -0,0 +1,21 @@
+{
+  "images" : [
+    {
+      "filename" : "Group 3.pdf",
```

Comment:
Next time, please ensure that the image name and file name are the same. To achieve this, please name the image correctly before importing it.

same for other images